sage.py

```diff
@@ -1291,13 +300,13 @@ def make_attack_graphs(victim_episodes, state_sequences, sev_sinks, datafile, di
 state_sequences = make_state_sequences(episode_subsequences, state_traces)
 
 # print('------ Clustering state groups ------')
-# state_groups = make_state_groups(state_sequences, path_to_traces)
+# state_groups = plot_state_groups(state_sequences, path_to_traces)
 
 print('------ Grouping episodes per (team, victim) ------')
-episodes_per_attacker, episodes_per_victim = group_episodes_per_av(state_sequences)
+episodes_per_victim, _ = group_episodes_per_av(state_sequences)
```

Comment:
Swapped them, for consistency (the method creates first victim, then attacker, also attacker dictionary does not have to be stored as a variable).

Depending on your reply to my comment above about `group_episodes_per_av`, this might not even be needed

Reply:
Sure. They can just be calculated in group_episodes_per_av() and returned. No need to use them in lines 306 onward.